data/migrations/V0087__operations_log_mv.sql

```diff
@@ -0,0 +1,48 @@
+-- Materialized View: public.ofec_operations_log_mv
+
+DROP MATERIALIZED VIEW IF EXISTS public.ofec_operations_log_mv;
+
+SET search_path = public;
+
+CREATE MATERIALIZED VIEW public.ofec_operations_log_mv AS 
```

Comment:
With the concern about too many materialized views, have we confirmed that one is actually needed?

Reply:
The request came from @jwchumley, since it was not easy to get the form_types from the existing operations_log view.

Reply:
I meant: "Would a _plain_ view suffice instead of a _materialized_ view"?

Reply:
@jwchumley @ChingKuan Could either one of you respond to this?

Reply:
Either one is good, only less than 200 msec difference in performance.  View is easy to maintain and no disk spaces request.  I've created a view (public.ofec_operations_log_vw) , you can try it.
--
select * from public.ofec_operations_log_vw where cand_cmte_id = 'C00329920'; -- execution time 198.6 mesc. 
--
select * from public.ofec_operations_log_mv where cand_cmte_id = 'C00329920'; -- execution time 0.38 mesc.